File: 프로그래머스/Level.1/18. 나누어 떨어지는 숫자 배열.js
function solution(arr, divisor) {
  var answer = [];

  for (let i = 0; i < arr.length; i++) {
    if (arr[i] % divisor == 0) {
      answer.push(arr[i]);
    }
  }

  return answer.length == 0 ? [-1] : answer.sort((a, b) => a - b);
}

solution([2, 36, 1, 3], 1);
solution([3, 2, 6], 10);

/////// 다른 풀이
function solution(arr, divisor) {
  var answer = arr.filter((v) => v % divisor == 0);
  return answer.length == 0 ? [-1] : answer.sort((a, b) => a - b);
}

File: 프로그래머스/Level.1/18. 나누어 떨어지는 숫자 배열.py
def solution(arr, divisor):
    num = []
    for i in arr:
        if (i % divisor == 0):
            num.append(i)

    return ([-1] if len(num) == 0 else sorted(num))

# 팀 코드
def solution(arr, divisor):
    newArr=[]
    for i in arr:
        if(i%divisor==0):
            newArr.append(i)
    if(len(newArr) == 0):
        newArr.append(-1)
    newArr.sort()
    return newArr